src/index.ts

```diff
@@ -71,6 +71,20 @@ export interface ProjectBuildOptions {
   js?: {minify?: boolean, compile?: boolean};
 }
 
+export interface LintOptions {
+  /**
+   * The lint rules to run. Can be the code of a collection of rules like
+   * "polymer-2.0" or an individual rule like "dom-module-invalid-attrs".
+   */
+  rules: string[];
+
+  /**
+   * Warnings to ignore. After the rules are run, any warning that matches
+   * one of these codes is ignored, project-wide.
+   */
+  ignore: string[];
```

Comment:
Might need a more specific name. At first read, I'd think this was files to ignore.